src/main/java/com/nike/vault/client/VaultClient.java

```diff
@@ -303,6 +318,7 @@ protected Response execute(final HttpUrl url, final String method, final Object
         try {
             return gson.fromJson(response.body().string(), typeOf);
         } catch (IOException|JsonSyntaxException e) {
+            logger.error("parseResponseBody: response={}", gson.toJson(response.body()));
```

Comment:
This will print the byte array input stream values, aka a bunch of gibberish. use response.body().string() and avoid needing gson altogether

Something like the following

`log.info("Received status code: {}, with Response Body: {}", response.code(), response.body().string())`